### PR TITLE
fixed documentation for docs-agent demo

### PR DESCRIPTION
**Pull request by thesanju**
## Description of the change
fixed bug in documentation which was causing a issue while running the Docs Agent chat app.

## Motivation
this bug was causing me issue https://github.com/google/generative-ai-docs/issues/176 and I added a extra line in documentation which solves this issue and @nickvander suggested me to fix this in documentation.

## Type of change
Documentation

## Checklist

- [x] I have performed a self-review of my code.
- [x] I have added detailed comments to my code where applicable.
- [x] I have verified that my change does not break existing code.
- [x] My PR is based on the latest changes of the main branch (if unsure, please run `git pull --rebase upstream main`).
- [x] I am familiar with the [Google Style Guide](https://google.github.io/styleguide/) for the language I have coded in.
- [x] I have read through the [Contributing Guide](https://github.com/google/generative-ai-docs/blob/main/CONTRIBUTING.md) and signed the [Contributor License Agreement](https://cla.developers.google.com/about).

### CODE REVIEW

File: demos/palm/python/docs-agent/README.md [hunk 1]
```diff
@@ -624,7 +628,7 @@ To launch the Docs Agent chat app, do the following:
    Notice the line that shows the URL of this server (`http://example.com:5000` in
    the example above).
 
-3. Open the URL above on a browser.
```
[the remark this answers is not in the document]
Comment (kyolee415) [reply demos/palm/python/docs-agent/README.md:627]: FYI, without having to close this existing pull request, you can just update changes to your local, forked repo and create a pull request again, which will update the changes in this pull request. Thank you!

[the remark this answers is not in the document]
Comment (thesanju) [reply demos/palm/python/docs-agent/README.md]: yes sorry I fixed it, you are the most patience person I ever seen.

**Review by kyolee415 — APPROVED** [review]
Finally! Thank you for this update! :)

**Comment by thesanju** [comment]
> Finally! Thank you for this update! :)

I don't know much about merging PR, can you tell me how it happens and when my PR will be merged? am just asking out of curies

**Comment by kyolee415** [comment]
Ah, I thought for some reason you were able to merge this pull request yourself, but I just realized that it might not be the case. :) I'm merging this pull request now. Thank you again!

**Pull request merged**